source/meetings/2016/november/index.html.md

```diff
@@ -0,0 +1,97 @@
+---
+updated_by:
+  email: murray.steele@gmail.com
+  name: Murray Steele
+created_by:
+  email: murray.steele@gmail.com
+  name: Murray Steele
+category: meeting
+title: November 2016 Meeting
+updated_at:
+published_at: 2016-10-17 13:20:16 +0100
+created_at: 2016-10-17 12:54:32 +0100
+parts: {}
+status: Published
+---
+
+The November 2016 meeting of LRUG will be on *__Tuesday__ the 15th of November*, from
+_6:00pm_ to _8:00pm_ (talks start at _6:30pm_).  The venue, [Code
+Node](https://skillsmatter.com/locations/264-skills-matter-codenode) between
+Moorgate and Liverpool St. stations, is provided by [Skills
+Matter](http://www.skillsmatter.com).  [Full venue and registration details
+are given below](#nov16registration).
+
+Agenda
+------
+
+### The invisible cost of code
+
+[Luke Morton](https://twitter.com/lukemorton) says:
+
+> Every line of code has cost associated with it. The cost of writing it. The
+> cost of testing it. Deploying it. Reading it. Changing it later. I'd like to
+> identify some of the costs and suggest ways of reducing them. Hint: write
+> less code in the first place.
+
+### AWS Elastic Beanstalk & Docker for Rails developers.
+
+[Tomas Valent](https://twitter.com/equivalent8) says:
+
+> Elastic Beanstalk (EB) is a product from AWS that is trying to provide easy
+> to setup load balanced environment of EC2 instances.
+>
+> In this talk we will configure simple Rails app, wrap it in a Docker container
+> and ship it to AWS EB setup. We will also play around with some configuration
+> options in order to squeeze maximum out of it.
```

Comment:
I think it's maximum "I just paste in what the speaker said verbatim", I dunno. ("performance" maybe?)